monitoring/mongodb/alerts.test.yaml

```diff
@@ -333,23 +333,32 @@ tests:
-        values: 0x10
-      - series: mongodb_rs_members_state{namespace="zenko", rs_nm="rs0", pod="data-db-mongodb-sharded-mongos-2", member_state="SECONDARY"}
+      - series: mongodb_rs_members_state{namespace="zenko", pod="data-db-mongodb-sharded-shard0-data-0", member_state="PRIMARY", rs_nm="data-db-mongodb-sharded-shard-0", member_idx="data-db-mongodb-sharded-shard0-data-0.data-db-mongodb-sharded-headless.zenko.svc.cluster.local:27017"}
+        values: 1x10
+      - series: mongodb_rs_members_state{namespace="zenko", pod="data-db-mongodb-sharded-shard0-data-0", member_state="SECONDARY", rs_nm="data-db-mongodb-sharded-shard-0", member_idx="data-db-mongodb-sharded-shard0-data-1.data-db-mongodb-sharded-headless.zenko.svc.cluster.local:27017"}
+        values: 2x10
+      - series: mongodb_rs_members_state{namespace="zenko", pod="data-db-mongodb-sharded-shard0-data-0", member_state="SECONDARY", rs_nm="data-db-mongodb-sharded-shard-0", member_idx="data-db-mongodb-sharded-shard0-data-2.data-db-mongodb-sharded-headless.zenko.svc.cluster.local:27017"}
+        values: 2x10
+      - series: mongodb_rs_members_state{namespace="zenko", pod="data-db-mongodb-sharded-shard0-data-1", member_state="PRIMARY", rs_nm="data-db-mongodb-sharded-shard-0", member_idx="data-db-mongodb-sharded-shard0-data-0.data-db-mongodb-sharded-headless.zenko.svc.cluster.local:27017"}
+        values: 1x10
+      - series: mongodb_rs_members_state{namespace="zenko", pod="data-db-mongodb-sharded-shard0-data-1", member_state="SECONDARY", rs_nm="data-db-mongodb-sharded-shard-0", member_idx="data-db-mongodb-sharded-shard0-data-1.data-db-mongodb-sharded-headless.zenko.svc.cluster.local:27017"}
+        values: 2x10
+      - series: mongodb_rs_members_state{namespace="zenko", pod="data-db-mongodb-sharded-shard0-data-1", member_state="SECONDARY", rs_nm="data-db-mongodb-sharded-shard-0", member_idx="data-db-mongodb-sharded-shard0-data-2.data-db-mongodb-sharded-headless.zenko.svc.cluster.local:27017"}
+        values: 2x10
+      - series: mongodb_rs_members_state{namespace="zenko", pod="data-db-mongodb-sharded-shard0-data-2", member_state="PRIMARY", rs_nm="data-db-mongodb-sharded-shard-0", member_idx="data-db-mongodb-sharded-shard0-data-0.data-db-mongodb-sharded-headless.zenko.svc.cluster.local:27017"}
         values: 1x10
-  
+      - series: mongodb_rs_members_state{namespace="zenko", pod="data-db-mongodb-sharded-shard0-data-2", member_state="SECONDARY", rs_nm="data-db-mongodb-sharded-shard-0", member_idx="data-db-mongodb-sharded-shard0-data-1.data-db-mongodb-sharded-headless.zenko.svc.cluster.local:27017"}
+        values: 2x10
+      - series: mongodb_rs_members_state{namespace="zenko", pod="data-db-mongodb-sharded-shard0-data-2", member_state="(not reachable/healthy)", rs_nm="data-db-mongodb-sharded-shard-0", member_idx="data-db-mongodb-sharded-shard0-data-2.data-db-mongodb-sharded-headless.zenko.svc.cluster.local:27017"}
+        values: 2 _ _ _ _ _ _ _ _ _
```

Comment:
when this serie disappears, the other series will actually change as well: as the remaining pods will not report on its status anymore... and the state reporte looks strange :/
* On "startup", we should have series : everything is running fine (3 pods having the status of 3 members)
* Then one pod "crashes" : the 3 associated series stop
* Then the other pods update their "vision" of the cluster : they either report a different member state (not reachable?) or drop the member completely (not sure exactly what happens)

Reply:
from what I saw the state is (not reachable/healthy) , the test was updated accordingly

Reply:
every metric from `pod="data-db-mongodb-sharded-shard0-data-2"` should be removed when we simulate the pod crashing...

so i guess time series should look something like this:

```
# First pod
- series: member_state{pod="data-0", state="PRIMARY", member_idx="data-0"}
   values: 1x10 1x10
- series: member_state{pod="data-0", state="SECONDARY", member_idx="data-1"}
   values: 2x10 2x10
- series: member_state{pod="data-0", state="SECONDARY", member_idx="data-2"}
   values: 2x10 stale      # `stale` (i.e. not present) after the 10s sample
- series: member_state{pod="data-0", state="(not reachable/healthy)", member_idx="data-2"}
   values: _x10 8x10      # appears from the 11s sample
# Second pod
- series: member_state{pod="data-1", state="PRIMARY", member_idx="data-0"}
   values: 1x10 1x10
- series: member_state{pod="data-1", state="SECONDARY", member_idx="data-1"}
   values: 2x10 2x10
- series: member_state{pod="data-1", state="SECONDARY", member_idx="data-2"}
   values: 2x10 stale      # `stale` (i.e. not present) after the 10s sample
- series: member_state{pod="data-1", state="(not reachable/healthy)", member_idx="data-2"}
   values: _x10 8x10      # appears from the 11s sample
# Third pod : stops responding after 10th sample
- series: member_state{pod="data-2", state="PRIMARY", member_idx="data-0"}
   values: 1x10 stale
- series: member_state{pod="data-2", state="SECONDARY", member_idx="data-1"}
   values: 2x10 stale
- series: member_state{pod="data-2", state="SECONDARY", member_idx="data-2"}
   values: 2x10 stale
```

Reply:
Providing this :  series: member_state{pod="data-0", state="SECONDARY", member_idx="data-2"}
   values: 2x10 stale      # `stale` (i.e. not present) after the 10s sample
   
  will cause the expression to return 2 => expected number of secondaries as it's not taking the state="(not reachable/healthy)" metric

Reply:
AFAIU, `stale` removes the metric series, which should what happens:
* the serie with state label equal to `secondary` (and value `2`) stops
* another serie starts with state label equal to `(not reachable/healthy)` and value `8`

(since one label value changes, this is really another serie : even if [as human] we think it is the same one...)